### DISCUSSION TITLE
Hiden Tabs and show on new tab created

### DISCUSSION THREAD

**Discussion by AndreM222** [ghostty-org/ghostty#3654] [Feature Requests, Ideas] [answered]
Hi,

First of all, incredible job. I have used it only for 2 days and I love it so far. Better experience I ever had

Second, I will like to recommend a tabs feature that ITerm2 has. On iterm you can have a kind of compact hidden title tabs that it does not show the closing button and other stuff like normal hidden does, but when a tab is created pressint cmd+t then it makes tabs as the macos-style = tabs.

here is a preview.

### Before new tab

<img width="1695" alt="スクリーンショット 2024-12-28 14 14 57" src="https://github.com/user-attachments/assets/720291c2-ea65-4003-8eef-e967c3c887fa" />

### After new tab

<img width="1715" alt="スクリーンショット 2024-12-28 14 15 21" src="https://github.com/user-attachments/assets/a505f77a-5d35-44c2-8e6b-08690665c172" />

**Comment by mitchellh** [comment, marked as answer]
Thanks for this suggestion. Getting back a bit later to this, I'm going to reject this, not because the idea is bad, but because I think we should reassess tab styling when we ultimately create a custom tab bar. (iTerm2 uses a custom tab bar). We need a custom tab bar for many reasons, not just style, so we should just take a look at this again then.
